### ISSUE TITLE
Multi agent "partially observable" setting

### ISSUE THREAD

**Issue by LAS1520** [rte-france/Grid2Op#412]
I am very interested in "partially observable" setting in multi agent branch, when will you add it?

**Comment by BDonnot** [comment]
Hello,

The environment itself is already partially observable (you can't observe the future).

But I think you mean that a given agent can only observe part of the grid state and not all the grid. If so I think it's already possible, even though I did not add any docs nor examples (but right now I don't have time :-/)

By the way, how do you find the multi agent feature? Is it usable? Some things are missing?

**Comment by LAS1520** [comment]
Thanks for your reply. 
Yes, the "partially observable" refers to the setting in which a given agent can only observe part of the grid state and not all the grid. I have tried to check the observation space of a single agent, and I found that it is partial grid observation. But when I used function BoxGymObsSpace to process it, it would become the full grid observation.
And I'm sorry that because I have not done lots of experiments, the multiagent environment basically worked for me.
